debian/tests/haschangelog

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+
+"""Test if associated PR has a changelog entry for the changes.
+
+"""
+
+import os
+import re
+import sys
+
+from git import InvalidGitRepositoryError, Repo
+from github import Github
+from urllib.request import urlopen
```

Comment:
The `requests` package might be a little nicer here.  You'd have to add it as a dependency in `d/t/control` but then you wouldn't have to [explicitly decode the text content](http://requests.readthedocs.io/en/master/user/quickstart/#response-content).